components/ai/code/ai-bbe/bring-text-alive-with-openai-chats.md

```diff
@@ -1,4 +1,4 @@
 ---
-title: "Bring text alive with the chat, completions, edits, and moderation APIs of Azure and OpenAI"
-description: "Azure and OpenAI’s text manipulation APIs allow you to bring text alive and program them easily. Ballerina connectors for these APIs give you type safe, structured ways to build applications quickly."
----
\ No newline at end of file
+title: "Simple When You Want It. Powerful When You Need It."
+description: "Ballerina offers high-level abstractions to make AI integration easy and intuitive—but it doesn’t stop there. For advanced use cases, you get full access to model provider APIs through robust, type-safe connectors. This means you can go beyond the basics and tap into provider-specific features with confidence and clarity, all while staying in the same language and ecosystem."
```

Comment:
Bit too lengthy. the practice was to have blade text under 250 characters. check all the places.